package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name":  "applicationinsights-js",
-    "version":  "0.22.16",
+    "version":  "0.22.17",
     "description":  "Microsoft Application Insights JavaScript SDK",
     "main":  "JavaScriptSDK/AppInsightsModule",
-    "keywords": [
-        "browser performance monitoring",
-        "script errors",
-        "application insights",
-        "microsoft",
-        "azure"
-    ],
+    "keywords":  [
+                     "browser performance monitoring",
+                     "script errors",
+                     "application insights",
+                     "microsoft",
+                     "azure"
+                 ],
```

Comment:
Not intentional. Powershell script to update the version number did that. 
I'll check if I can fix it later today and submit a fix in a separate pull request.